### ISSUE TITLE
Make a myria connection-based catalog available

### ISSUE THREAD

**Issue by bmyerz** [uwescience/raco#411]
We want to encourage use of Raco for plan manipulations as much as possible. Since the MyriaL optimizer expects a catalog, we should provide one in Raco. This probably just involves extracting (the one in myria-web](https://github.com/uwescience/myria-web/blob/grappa-master-b/appengine/myria_catalog.py).

- [ ] access to MyriaCatalog
- [ ] add this catalog option to `scripts/myrial`
- [ ] @billhowe wanted serialization to `FromFileCatalog` format, too, so that catalogs can be cached locally

**Comment by billhowe** [comment]
Agreed.  Myria-web should be web stuff only: a thin REST wrapper, the GUI, viz.  For all else, it should be exercising RACO.
